src/com/t_oster/visicut/model/graphicelements/svgsupport/SVGImporter.java

```diff
@@ -237,22 +267,26 @@ public double determineResolution(File f, List<String> warnings)
     // SVG files loaded in inkscape retain the Illustrator comment, but are saved with 90 DPI.
     if (AdobeIllustratorSeen) { result = 72; }
     if (WwwInkscapeComSeen) { result = 90; }	// inkscape wins over Illustrator
+    if (InkscapeVersion092Seen) { result = 96; }	// inkscape with known version wins over anything else.
     if (result != 90)
     {
        if (AdobeIllustratorSeen)
          {
            warnings.add("Adobe Illustrator comment seen in SVG.");
 	 }
-       warnings.add("Switching to DPI from 90 to " + result + " - Please check object size!");
-
+       if (InkscapeVersion092Seen)
+         {
+           warnings.add("Inkscape Version 0.92+ comment seen in SVG.");
+	 }
+       warnings.add("Switching DPI from 90 to " + result + " - Please check object size!");
     }
     return result;
   }
 
   /*
    * Tries to determine the Coordinate resolution in DPI.
    * SVG default is 90, but AI generates 72??
```

Comment:
Actually, the SVG standard says **96**, but inkscape < 0.92 wrongly used 90 dpi. http://wiki.inkscape.org/wiki/index.php/Release_notes/0.92#Important_changes

We should somehow suppress the DPI warning message for files loaded from a recent inkscape version via the inkscape extension, because users will ignore a warning that pops up every time.